src/adapters/LocalStorageJoinProvider.ts

```diff
@@ -15,6 +15,23 @@ export default class LocalStoreageJoinProvider implements JoinStorageProvider {
     this.currentDatabase = null;
   }
 
+  getStorageAddress() {
+  }
+
+  connectToStorage() {
```

Comment:
I think maybe we should return something here cause eventually when we strongly type the interface (i.e. set the type signature for each function when doing jdocs) it will cause the linter to panic when you dont return anything here

Reply:
Modified to return voids